go/vt/orchestrator/config/config_test.go

```diff
@@ -15,104 +15,52 @@ func init() {
 func TestReplicationLagQuery(t *testing.T) {
 	{
 		c := newConfiguration()
-		c.SlaveLagQuery = "select 3"
-		c.ReplicationLagQuery = "select 4"
-		err := c.postReadAdjustments()
-		test.S(t).ExpectNotNil(err)
-	}
-	{
-		c := newConfiguration()
-		c.SlaveLagQuery = "select 3"
 		c.ReplicationLagQuery = "select 3"
 		err := c.postReadAdjustments()
 		test.S(t).ExpectNil(err)
 	}
-	{
-		c := newConfiguration()
-		c.SlaveLagQuery = "select 3"
-		c.ReplicationLagQuery = ""
-		err := c.postReadAdjustments()
-		test.S(t).ExpectNil(err)
-		test.S(t).ExpectEquals(c.ReplicationLagQuery, "select 3")
-	}
 }
 
 func TestPostponeReplicaRecoveryOnLagMinutes(t *testing.T) {
 	{
 		c := newConfiguration()
```

Comment:
I'm wondering if it's even worth keeping these tests. Their intent seems to be test copying of the value from the old param to the new one. Since we are deleting the old param there doesn't seem to be much left to test.

Reply:
agreed

Reply:
Removed